ursa/graph/graph.py

```diff
@@ -234,3 +244,13 @@ def getattr(self, item):
         @return: The attribute requested.
         """
         return getattr(self, item)
+
+    def clean_old_rows(self):
+        """Spills all rows older than versions_to_store to disk.
+        """
+        for k, rows in self.rows.items():
+            if len(rows) < self.versions_to_store:
+                continue
+            self.rows[k] = rows[-self.versions_to_store:]
+            rows_to_write = rows[:-self.versions_to_store]
+            [write_vertex(r, self.graph_id, k) for r in rows_to_write]
```

Comment:
`write_vertex` returns the path, but this task invocation does not store it.

Reply:
Resolved.